plugins/ruby-foreman-templates/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-foreman-templates (2.0.1) stable; urgency=low
```

Comment:
Add a -1 here, or use the scripts to do it automatically.  (The previous entry was a mistake.)